full_install.sh

```diff
@@ -0,0 +1,26 @@
+deactivate
+rm -rf ./venv
+python3 -m venv ./venv
```

Comment:
why make a new venv? I would expect it to install on whatever environment is currently in use by the user.

Reply:
We can remove this. I was primarily using this script for development.